python/paddle/fluid/layers/io.py

```diff
@@ -846,14 +851,25 @@ def network(img, label):
          img, label = fluid.layers.read_file(reader)
          loss = network(img, label)  # some network definition
 
-         fluid.Executor(fluid.CUDAPlace(0)).run(fluid.default_startup_program())
-
-         exe = fluid.ParallelExecutor(use_cuda=True, loss_name=loss.name)
-         for epoch_id in range(10):
+         place = fluid.CUDAPlace(0) if USE_CUDA else fluid.CPUPlace()
+         exe = fluid.Executor(place)
+         exe.run(fluid.default_startup_program())
+
+         exec_strategy = fluid.ExecutionStrategy()
+         exec_strategy.num_threads = DEV_COUNT * 4 # the size of thread pool.
```

Comment:
line 858~859 should be removed, because the default value of exec_strategy.num_threads is DEV_COUNT * 4.